client/src/App.js

```diff
@@ -103,7 +103,7 @@ function App() {
     history.push('usersPage')
   }
 
-  function deleteFetch(id) {
+  function withoutDeletedUserFetch(id) {
```

Comment:
I would suggest naming your function with a verb and noun so it becomes clearer what the function exactly does.